Explorer/Assets/Scripts/Global/Dynamic/DynamicSceneLoader.cs

```diff
@@ -166,12 +168,9 @@ async UniTask ChangeRealmAsync(string selectedRealm, CancellationToken ct)
 
                 await UniTask.SwitchToMainThread();
 
-                Vector3 characterPos = ParcelMathHelper.GetPositionByParcelPosition(settings.StartPosition);
-                characterPos.y = 1f;
-
-                staticContainer!.CharacterObject.Controller.transform.position = characterPos;
-
-                await realmController.SetRealmAsync(URLDomain.FromString(selectedRealm), ct);
+                var loadReport = new AsyncLoadProcessReport(new UniTaskCompletionSource(), new AsyncReactiveProperty<float>(0));
+                dynamicWorldContainer.MvcManager.ShowAsync(SceneLoadingScreenController.IssueCommand(new SceneLoadingScreenController.Params(loadReport))).Forget();

```

Comment:
Minimize the number of flows that you launch and forget. it leads to losing control. Merge two async processed into `await UniTask.WhenAll(...)`